project_template/BUILDING.md

```diff
@@ -0,0 +1,11 @@
+
+## Requirements
+
+This Azure SDK module has several external build-time requirements.
+
+- **Catch2** (for tests): https://github.com/catchorg/Catch2.git

```

Comment:
>tests [](start = 18, length = 5)

catch2 doesn't seem to come with a mocking framework. What's the story around that requirement?

Reply:
the story is "we haven't decided on a test framework, I wanted a test example, the buildsystem integration is almost identical for all/most test frameworks, and I had a project using catch2 open on my other monitor"